releases/release-1.17/README.md

```diff
@@ -0,0 +1,196 @@
+# Kubernetes 1.17
+
+#### Links
+* [This document](https://git.k8s.io/sig-release/releases/release-1.17/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.17/release_team.md)
+* [Meeting Minutes](http://bit.ly/k8s117-releasemtg)
+* [Kubernetes Release Calendar](https://bit.ly/k8s-release-cal)
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](http://bit.ly/k8s116-enhancement-tracking) TODO
+* [Bug Triage Tracking Sheet](http://bit.ly/k8s116-bugtriage) TODO
+* [CI Signal Report](http://bit.ly/k8s116-cisignal) TODO
+* [Retrospective Document](http://bit.ly/k8s117-retro)
+* [kubernetes/kubernetes v1.17 milestone](https://github.com/kubernetes/kubernetes/milestone/43)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## tl;dr
+The 1.17 release cycle is proposed as follows:
+- **Monday, September 23**  - Release Cycle Begins
+- **Tuesday, October 15, EOD PST** - Enhancements Freeze
+- **Thursday, November 14, EOD PST**  - Code Freeze
+- **Tuesday, November 19** - Docs must be completed and reviewed
+- **Monday, December 9** - Kubernetes 1.16.0 Released
+
+## What will we do differently this release?
+* Due to the end of the year, this release cycle will be shorter than the other cycles this year. In accordance, we will have a slightly accelerated schedule.
+
+## What are we continuing from the previous release
+* All Enhancements must have a KEP that is in `implementable` state by Enhancements Freeze. If the enhancement does not have a KEP in `implementable` state by Enhancements Freeze it will be removed from the milestone and will require an [exception](/releases/EXCEPTIONS.md).
+* The Emeritus Adviser will be responsible for advising the Release Team Lead and Lead Shadows, ensuring shadow selection happens, and that release shadows are getting the most out of their participation in the release process.
+* The itemized list of code changes for the release is being migrated from the release changelog `CHANGELOG-x.y.md` (where `x` and `y` are major and minor versions of Kubernetes, respectively) to [https://relnotes.k8s.io](https://relnotes.k8s.io)
+* We will continue to implement guidelines and clean up around CI Signal.
+
+## Timeline
+| **What** | **Who** | **Sept** | **Oct** | **Nov** | **Dec** | **WEEK** | **CI SIGNAL** |
+| --- | --- | --- | --- | --- | --- | --- | --- |
+| Team Leads finalized | Lead | Fri 20 | | | | week 0 | |
+| Start of Release Cycle | Lead | Mon 23 | | | | week 1 | [master-blocking] |
+| Schedule finalized | Lead | Wed 25 | | | | | |
+| Team Shadows finalized | Lead | Wed 25 | | | | | |
+| Start Enhancements Tracking | Enhancements Lead | | Tue 1 | | | week 2 | |
```

Comment:
Should/can enhancements tracking start early?  I wonder if only two weeks is enough before enhancements freeze for SIG outreach regarding their plans, especially socializing the idea that this is the Q4 release on a tight schedule so SIGs probably needs to think more about stability than enhancements or constraining to lower risk enhancements?

Reply:
We had two weeks between tracking start and freeze in 1.16, a non-tightly scheduled release. I have yet to speak to @mrbobbytables but the enhancements handboock has this as a less strictly defined process anyway, so I'm not sure how much value the tracking spreadsheet has.
I'm planning to send out an email to sig leads as soon as this is finalized.

Reply:
+1 on starting tracking early. With KubeCon NA this release I'm definitely in favor of approaching the edge of **over-communicating** the deadlines. It's going to be a busy time for many and things will be easy to miss.

Reply:
reflected in schedule! @tpepper  - PTAL <3